pkg/apis/config/features.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	cm "knative.dev/pkg/configmap"
+)
+
+type Flag string
+
+const (
+	// FeaturesConfigName is the name of the ConfigMap for the features.
+	FeaturesConfigName = "config-features"
+
+	Enabled  Flag = "Enabled"
+	Allowed  Flag = "Allowed"
+	Disabled Flag = "Disabled"
+)
+
+func defaultFeaturesConfig() *Features {
+	return &Features{
+		MultiContainer: Disabled,
+	}
+}
+
+// NewFeaturesConfigFromMap creates a Features from the supplied Map
+func NewFeaturesConfigFromMap(data map[string]string) (*Features, error) {
+	nc := defaultFeaturesConfig()
+
+	if err := cm.Parse(data,
+		AsFlag("multi-container", &nc.MultiContainer),
+	); err != nil {
+		return nil, err
+	}
+	return nc, nil
```

Comment:
More readable IMO. The line is not _that_ long.
```suggestion
	if err := cm.Parse(data, AsFlag("multi-container", &nc.MultiContainer)); err != nil {
		return nil, err
	}
	return nc, nil
```